business_requirement_deliverable_project/models/business.py

```diff
@@ -42,6 +43,36 @@ def generate_projects_wizard(self):
         string='Total Planned Hour in RL related to business requirement',
         compute='_compute_planned_hour'
     )
+    linked_project_count = fields.Integer(
+        compute='_compute_linked_project_count',
+        string="Number of Business Requirements"
+    )
+
+    @api.depends('linked_project', 'deliverable_lines')
+    def _compute_linked_project_count(self):
+        for rec in self:
+            domain = ['|',
```

Comment:
@darshan-serpent @YogeshMahera-SerpentCS Need to understand the exactly purpose of this domain?
This two fields:
`business_requirement_id`
`business_requirement_deliverable_id`

As parameters domains means should be fields on the `business.requirement` model, right?